### PR TITLE
Uplift translations from main to release branches

### PR DESCRIPTION
**Pull request by ionutbedregeanu**
For #5896 
No unit tests, screenshots, accessibility required
### Pull Request checklist

- [x] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [x] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not
- [x] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features. In addition, it includes a screenshot of a successful [accessibility scan](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=en_US) to ensure no new defects are added to the product.

### To download an APK when reviewing a PR:
1. click on `Show All Checks`,
2. click `Details` next to `build-focus-debug` or `build-klar-debug` for changes targeting Klar,
2. click `View task in Taskcluster`,
3. click the `Artifacts` row,
4. click to download any of the apks listed here which use an appropriate name for each CPU architecture.

### CODE REVIEW

File: .github/workflows/sync-strings.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    - cron: '0 2 * * *'
+
+jobs:
+  main:
+    name: "Sync Strings"
+    runs-on: ubuntu-20.04
+    steps:
+      - name: "Discover Focus Beta Version"
+        id: focus-android-beta-version
+        uses: mozilla-mobile/focus-android-beta-version@2.0.0
```
Comment (Mugurell) [inline .github/workflows/sync-strings.yml:18]: This is the only thing that seems to be missing. The script hardcodes the Fenix project so it cannot be used directly.
Looked into adding a new repo similar to https://github.com/mozilla-mobile/fenix-beta-version for Focus also but going through that code it seems we can adapt that project to be a general one that can be used for many other Github actions from Mozilla or others and opened a new ticket for that - https://github.com/mozilla-mobile/fenix-beta-version/issues/2.

**Review by Mugurell — CHANGES_REQUESTED** [review]
This looks good but there is a dependency missing.
Waiting for Aki's response on how to support getting the beta version.

File: .github/workflows/sync-strings.yml (hunk 1 above)
Comment (escapewindow) [reply .github/workflows/sync-strings.yml:18]: Let's use https://github.com/mozilla-mobile/fenix-beta-version/releases/tag/3.0.0 .

File: .github/workflows/sync-strings.yml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+          src: main
+          dst: beta
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v3
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          path: beta
+          branch: automation/sync-strings-${{ steps.focus-android-beta-version.outputs.beta_version }}
+          title: "Sync Strings from main to releases_${{steps.focus-android-beta-version.outputs.beta_version}}.0"
+          body: "This (automated) PR syncs strings from `main` to `releases_${{steps.focus-android-beta-version.outputs.focus-android-beta-version}}.0.0`"
```
Comment (Mugurell) [inline .github/workflows/sync-strings.yml:44]: One small detail: The output here should be just "beta_version" as above. 
And as a small style details spaces before and after accolades could be added also for consistency with the above code.
```suggestion
          body: "This (automated) PR syncs strings from `main` to `releases_${{ steps.focus-android-beta-version.outputs.beta_version }}.0.0`"
```

**Review by Mugurell — APPROVED** [review]
image

**Pull request merged**